src/QuantumSdk/README.md

```diff
@@ -79,7 +79,14 @@ When a rewrite steps fails to execute, setting the `QscVerbosity` to "Detailed"
 ```
   <QscVerbosity>Detailed</QscVerbosity>
 ```
-By default, the compiler will search the project output directory for a suitable assembly in case a dependency cannot be found. In case of a `FileNotFoundException`, and option may be to add the corresponding reference to the project, or to copy the missing dll to the output directory. 
+A `FileNotFoundException` will be raised if a compiler extension attempts to load a reference that either could not be found, or could not be loaded for other reasons. 
+By default, the compiler will search the project output directory for a suitable assembly in case a dependency cannot be found.
+If such an exception occurs during a compilation step loaded from a package reference, adding the package containing the dll that could not be found to the project or to copying the missing dll to the output directory may resolve the issue. 
```

Comment:
After the comma, this line is confusing. What are you supposed to do if the exception occurs? Consider starting after the comma with 'the issue may be resolved by...' rather than putting that piece at the end.

Reply:
Consider replacing 'dll that could not be found' with 'missing dll'.

Reply:
Yes, that makes it better. Thanks!